src/features/locators/LocatorsPage.tsx

```diff
@@ -157,15 +157,12 @@ export const LocatorsPage = () => {
         overlayClassName="jdn__button-tooltip"
         title={isDisabled ? 'Please select locators for your current page object.' : ''}
       >
-        <Button
-          ref={saveLocatorsRef}
-          type="primary"
-          onClick={handleConfirm}
-          className="jdn__buttons"
-          disabled={isDisabled}
-        >
-          {saveButtonLabel}
-        </Button>
+        {/* wrapper div below is needed to prevent loosing button ref during onboarding */}
```

Comment:
unnecessary comment

Reply:
Not sure. It’s a clarification for the div which looks redundant at the first sight and can be accidentally removed during some refactoring.

Reply:
Thanks for your comment. Having a ref inside a div indicates that it serves a purpose and is not redundant. Using separate blocks with ref is a common practice in React.